src/styles/signup-header.styles.css

```diff
@@ -0,0 +1,30 @@
+.signup-header {
+    width: 100%;
+    height: 65px;
+    margin-top: 35px;
+    flex-direction: row;
+    position: absolute;
+    font-family: 'Montserrat';
+    font-style: normal;
+}
+
+.left {
+    margin-left: 88px;
+}
+
+.comp-name {
+    position: absolute;
+    margin-top: 15px;
+    margin-left: 100px;
+}
+
+.page-title {
+    width: 452px;
```

Comment:
overall not bad, just change all exact pixel dimensions to vh and vw portions, as this will currently only work on your local display. 

<img width="1420" alt="Screen Shot 2022-06-26 at 4 27 57 PM" src="https://user-images.githubusercontent.com/75400743/175837956-a93cb741-04a5-434b-aeb1-de96c46b1a7a.png">